### PR TITLE
L1 muon online DQM uprades

### PR DESCRIPTION
**Pull request by thomreis**
Upgrade to the L1 muon online DQM.

Adding plots with ratio of muon and zero suppression mismatches. These are going to be used for quality tests.
Adding data to emulator comparisons for the uGMT intermediate muons.

### CODE REVIEW

**Comment by dmitrijus** [comment]
+1

File: DQM/L1TMonitorClient/src/L1TStage2RatioClient.cc [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+}
+
+void L1TStage2RatioClient::processHistograms(DQMStore::IGetter& igetter)
+{
+  const MonitorElement* numME_ = igetter.get(inputNum_);
+  const MonitorElement* denME_ = igetter.get(inputDen_);
+
+  if (numME_ && denME_) {
+    TH1F *hNum = numME_->getTH1F();
+    TH1F *hDen = dynamic_cast<TH1F *>(denME_->getTH1F()->Clone("den"));
```
Comment (davidlange6) [inline DQM/L1TMonitorClient/src/L1TStage2RatioClient.cc]: hi @thomreis - does the memory from this pointer not need to be cleaned up?

Comment (thomreis) [inline DQM/L1TMonitorClient/src/L1TStage2RatioClient.cc]: Hi @davidlange6 Yes indeed. Fixed.

**Comment by dmitrijus** [comment]
+1

**Comment by thomreis** [comment]
The failed tests do not seem to be related to the changes in the last commit. I find only file open errors.

**Comment by Martin-Grunewald** [comment]
please test

**Comment by davidlange6** [comment]
+1

**Pull request merged**